README.md

```diff
@@ -13,17 +13,28 @@ MERN is a scaffolding tool which makes it easy to build isomorphic apps using Mo
 - [Documentation](http://mern.io/documentation.html)
 - [Discussions](https://hashnode.com/n/mern)
 
+## Road to V3!
+We're on our way towards V3, which will bring a few updates to MERN! We are tracking with the following milestones:
+
+- V2.4.0 -> Webpack 3 and many fixes *Completed* : [list of changes](Changes.md)
+- V2.5.0 -> React V16.x upgrade and propTypes fix
+- V2.6.0 -> Webpack upgrade to V4
+- V2.7.0 -> database/mongo/mongoose updates/fixes
+- V2.8.0 -> React Router V4
+- V2.9.0 -> Express/Server dependencies/code upgrade
+- V3.0.0 -> CLI/devtools optimization (docker, etc.)
+
 ## Quickstart
 
-```
+```sh
   npm install -g mern-cli
   mern init your_new_app
   cd your_new_app
   npm install
   npm start
 ```
 
-**Note : Please make sure your MongoDB is running.** For MongoDB installation guide see [this](https://docs.mongodb.org/v3.0/installation/). Also `npm3` is required to install dependencies properly.
+**Note : Please make sure your MongoDB is running.** For MongoDB installation guide see [this](https://docs.mongodb.org/v3.0/installation/). Also `npm5` is required to install dependencies properly.
```

Comment:
Are we encouraging the use of npm at 5 or 6? `changes.md` says 6 :)

Reply:
@mattlubner let me know how you'd prefer to proceed related to npm version, stick to 5 or use 6.

Reply:
Just made the readme change.